tests/compat.rs

```diff
@@ -160,8 +169,16 @@ mod pgp {
             )?
             .epoch(1)
             .pre_install_script("echo preinst")
-            .add_changelog_entry("me", "was awesome, eh?", 123_123_123)
-            .add_changelog_entry("you", "yeah, it was", 12_312_312)
+            .add_changelog_entry(
+                "me",
```

Comment:
We should use the standard packaging guidelines for the author value, which means

`FirstName LastName <email> - VersionAndRelease`

The description should start with `-`

So for the changelog entry (in a specfile)

> %changelog
> \* Tue May 31 2016 Adam Miller <maxamillion@fedoraproject.org> - 0.1-1
> \- First bello package
> \- Example second

The date would go to date, "Adam Miller <maxamillion@fedoraproject.org> - 0.1-1" would go to "author", and 

> \- First bello package
> \- example second item in the changelog for version-release 0.1-1

would be the description

Reply:
I think that needs some more refinement, why would we make the version suffix `- 0.1-1` part of the author? It appears the API is not quite there and I'd like to carve that out in a separate PR.